src/setupProxy.js

```diff
@@ -6,10 +6,10 @@ module.exports = (app) => {
   // app.use('/token', createProxyMiddleware({ target: 'https://360Api.gordon.edu/', changeOrigin: true }));
 
   // @TRAIN
-  app.use('/api', createProxyMiddleware({ target: 'https://360ApiTrain.gordon.edu/', changeOrigin: true }));
-  app.use('/token', createProxyMiddleware({ target: 'https://360ApiTrain.gordon.edu/', changeOrigin: true }));
+  // app.use('/api', createProxyMiddleware({ target: 'https://360ApiTrain.gordon.edu/', changeOrigin: true }));
+  // app.use('/token', createProxyMiddleware({ target: 'https://360ApiTrain.gordon.edu/', changeOrigin: true }));
 
   // @LOCALHOST
-  // app.use('/api', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));
-  // app.use('/token', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));
+  app.use('/api', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));
+  app.use('/token', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));
```

Comment:
```suggestion
  // app.use('/api', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));
  // app.use('/token', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));
```

Reply:
Too soon, I literally just did that in the last commit because I am trying to test it right now